physionet-django/physionet/context_processors.py

```diff
@@ -9,3 +9,11 @@ def access_policy(request):
 
 def platform_name(request):
     return {'SITE_NAME': settings.SITE_NAME}
+
+
+def footer_data(request):
```

Comment:
Do we need a separate context processor for each variable? Is there a reason why we don't have a single `platform_config(request)` processor with SITE_NAME, FOOTER_MANAGED_BY, etc?

Reply:
I combined them.